internal/mage/util/engine.go

```diff
@@ -43,14 +43,14 @@ if [ -f /sys/fs/cgroup/cgroup.controllers ]; then
 	# move the processes from the root group to the /init group,
 	# otherwise writing subtree_control fails with EBUSY.
 	# An error during moving non-existent process (i.e., "cat") is ignored.
-	mkdir -p /sys/fs/cgroup/init
-	xargs -rn1 < /sys/fs/cgroup/cgroup.procs > /sys/fs/cgroup/init/cgroup.procs || :
+	rootlesskit mkdir -p /sys/fs/cgroup/init
```

Comment:
This line fails with error

```
2023-09-19 21:16:20 mkdir: can't create directory '/sys/fs/cgroup/init': Read-only file system
2023-09-19 21:16:20 [rootlesskit:child ] error: command [mkdir -p /sys/fs/cgroup/init] exited: exit status 1
2023-09-19 21:16:20 [rootlesskit:parent] error: child exited: exit status 1
```

I tried to wrap it around `rootlesskit` but I have the same issue, I do not know how to fix that :/

For now I cannot run dagger as a rootless container.